python/task/workflow.py

```diff
@@ -541,6 +541,14 @@ def write(self, producer, output, sharding=None, name=None, params=None):
 
     return output
 
+  def pipe(self, command, format=None, name=None):
```

Comment:
Awesome. Could very well come in handy in other situations as well as we move forward.

Reply:
Thanks for the speedy review.